packages/components/src/Select/Select.tsx

```diff
@@ -24,6 +24,7 @@ export type SelectProps = {
   disabled?: boolean;
   required?: boolean;
   error?: string;
+  width?: string;
```

Comment:
I would like to restrict the type here more, like responsive style values?

Reply:
Yes I thought about this. Like this at menuWidth: https://react-spectrum.adobe.com/react-spectrum/Picker.html#props
but we have not yet, right? We could add it to the BaseTheme?

Reply:
I think we have ResponsiveStyleValue which is used in other components as well :)

Reply:
oh sure, I did not think of that just now 🙈